docs/releases/release_checklist.md

```diff
@@ -47,3 +47,10 @@ This is a checklist for cutting a release
     * Create a fresh environment.
     * Pip install tern.
     * Run appropriate tests.
+
+- [ ] Prepare sources tarball.
+    * In the release environment, create a new directory called `vendor`
+    * Run `pip install -d vendor --require-hashes --no-binary :all: -r docs/releases/v1_0_0-requirements.txt`
```

Comment:
Do you want to change this to be more generic? i.e.
```
docs/releases/v<release>-requirements.txt
```

Reply:
Oops! Absolutely! Thanks for catching this.